config/MezaCoreExtensions.yml

```diff
@@ -542,7 +542,7 @@ list:
       $wgSDUUseJobQueue = false;
 
   # Html5mediator - Defines a new tag - html5media - which can be used to embed HTML5 video and/or audio.
```

Comment:
@todo Use Special:ReplaceText to search your wiki(s) for usage of `<html5media>` so that you can tag those pages with a FIXME-html5media category, and then work to fix broken content.

The **replacement** extension would be [Extension:EmbedVideo_(fork)](https://www.mediawiki.org/wiki/Extension:EmbedVideo_(fork))